src/host/libpixyusb_swig/get_frame.py

```diff
@@ -0,0 +1,100 @@
+#!/usr/bin/python
+import cv2
+from datetime import datetime
+import numpy as np
+import os
+import pixy
+import time
+
+"""
```

Comment:
Let's have doxygen comments for the file and functions.

Reply:
will do.

Reply:
Done